support-frontend/assets/helpers/productPrice/billingPeriods.ts

```diff
@@ -1,58 +1,156 @@
-const Annual = 'Annual';
-const Monthly = 'Monthly';
-const Quarterly = 'Quarterly';
+import type { ContributionType } from 'helpers/contributions';
+import type { ActiveRatePlanKey } from 'helpers/productCatalog';
 
-export type BillingPeriod = typeof Annual | typeof Monthly | typeof Quarterly;
-
-export type WeeklyBillingPeriod =
-	| typeof Monthly
-	| typeof Quarterly
-	| typeof Annual;
-
-const weeklyBillingPeriods = (): WeeklyBillingPeriod[] => {
-	return [Monthly, Quarterly, Annual];
-};
+export enum BillingPeriod {
+	Annual = 'Annual',
+	Monthly = 'Monthly',
+	Quarterly = 'Quarterly',
+	OneTime = 'One_Off',
```

Comment:
Tiny thing (for a future PR) - but should we pick either "one time" or "one off" as the way to describe this rather than mix them here?

Reply:
Have made it `OneTime` to match for the moment, as its an enum we can change this relatively easily in future if you feel its more readable with a space in it. Updated.